runtime/src/main/java/org/corfudb/runtime/Restore.java

```diff
@@ -84,30 +93,44 @@ public void start() throws IOException {
     }
 
     private void restore() throws IOException {
-        for (UUID streamId : streamIDs) {
-            String fileName = restoreTempDirPath + File.separator + streamId;
+        if (clearAllTablesBeforeRestore) {
+            clearAllTables();
+        }
+
+        long startTime = System.currentTimeMillis();
+        for (String tableBackup : tableBackups) {
+            UUID streamId = UUID.fromString(tableBackup.substring(0, tableBackup.indexOf(".")));
```

Comment:
would be great to move this into `tableBackup`, like: 
```
UUID streamId = tableBackup.getStreamID()
```

Reply:
sorry, `tableBackup` is a string, then question - why we keep strings instead of UUID-s?

Reply:
`tableBackups` is the file names of what comes out from unpacking the backup tar file, which contains the data of each table that was previously backed up. A tableBackup is more than just UUID, it's actually `uuid.namespace$tableName`. I noticed that I didn't update the comment above this variable so that caused your confusion.